code/modules/cargo/packs.dm

```diff
@@ -1897,33 +1897,68 @@
 	crate_name = "standard costume crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
-/datum/supply_pack/costumes_toys/randomised/toys
-	name = "Toy Crate"
-	desc = "Who cares about pride and accomplishment? Skip the gaming and get straight to the sweet rewards with this product! Contains five random toys. Warranty void if used to prank research directors."
-	cost = 5000 // or play the arcade machines ya lazy bum
-	// TODO make this actually just use the arcade machine loot list
-	num_contained = 5
-	contains = list(/obj/item/toy/spinningtoy,
-	                /obj/item/toy/sword,
-	                /obj/item/toy/foamblade,
-	                /obj/item/toy/talking/AI,
-	                /obj/item/toy/talking/owl,
-	                /obj/item/toy/talking/griffin,
-	                /obj/item/toy/nuke,
-	                /obj/item/toy/minimeteor,
-	                /obj/item/toy/plush/carpplushie,
+ /datum/supply_pack/costumes_toys/randomised/toys
+ 	name = "Toy Crate"
+ 	desc = "Who cares about pride and accomplishment? Skip the gaming and get straight to the sweet rewards with this product! Contains five random toys. Warranty void if used to prank research directors."
+ 	cost = 5000 // or play the arcade machines ya lazy bum
+ 	// TODID make this actually just use the arcade machine loot list
+ 	num_contained = 10
```

Comment:
Ten Items or Five, VOTE
👍 = 10 Items
🎉 = 5 Items

Reply:
actually, each item in amount of 10, or just 10 random items in amount of 1?

Reply:
the latter, don't scare me.

Reply:
Closing the vote on Fri Nov 9th

Reply:
Please don't vote for 10. We don't need to have every toy in every crate every time.